scout/build/variant/variant.py

```diff
@@ -210,6 +215,25 @@ def build_variant(
     if "str_pathologic_min" in variant:
         variant_obj["str_pathologic_min"] = variant["str_pathologic_min"]
 
+    if "str_swegen_mean" in variant:
+        variant_obj["str_swegen_mean"] = (
+            float(variant["str_swegen_mean"]) if variant["str_swegen_mean"] else None
+        )
+
+    if "str_swegen_std" in variant:
+        variant_obj["str_swegen_std"] = (
+            float(variant["str_swegen_std"]) if variant["str_swegen_std"] else None
+        )
+
+    if "str_inheritance_mode" in variant:
+        variant_obj["str_inheritance_mode"] = variant["str_inheritance_mode"]
+
+    if "str_disease" in variant:
+        variant_obj["str_disease"] = variant["str_disease"]
+
+    if "str_source" in variant:
```

Comment:
Wasn't this one called "str_source_display" when you initialize it before, or is it another one?

Reply:
str_source is the dict, but yeah, I got the docstring wrong! 👍